### DISCUSSION TITLE
Runtime.config.json not found

### DISCUSSION THREAD

**Discussion by AshwinSridharan0410** [paketo-buildpacks/feedback#228] [.NET Core Team]
I am running a dotnet 6.0 application. The build was successful. Before the exporting step, I am getting this error
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/5aca25f1-ef86-4040-b435-1588698a95a6)

paketo buildpack for asp.net core runtime 0.3.20 skipping build process. no runtimeconfig.json found. paketo buildpack for .net execute 0.14.17 failed to find *.runtimeconfig.json: no *.runtimeconfig.json found: file does not exist

**Comment by sophiewigmore** [comment]
What do you mean the build was successful? Can you post the full the logs of the build including the command you ran? That would be helpful

Did your build ever work with a different version of the buildpacks, or is this your first time trying to build it?

**Comment by AshwinSridharan0410** [comment]
Ohh I am sorry, the build failed actually. 
I went through the .go files written for dotnet-execute and dotnet-asp-runtime.
The build should happen even if there is no runtimeconfig.json is present , right ?
Let me share some snippets
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/5ee793ab-2e0d-400d-b4cb-6d530ad4d2ae)
The above one is of paketo-buildpacks/dotnet-execute
And this link will just explain the detect file of dotnet-core-asp-runtime 
https://github.com/paketo-buildpacks/dotnet-core-aspnet-runtime/blob/main/detect.go (It checks for framework-version inside runtimeconfig.json)
I am specifying the projectPath too with BP_DOTNET_PROJECT_PATH
And the application that I am trying to build is .net6 with azure functions. Does this has any implications along with it?
 I have built .net apps without runtimeconfig.json and it worked fine.

And for this question raised, 
Did your build ever work with a different version of the buildpacks, or is this your first time trying to build it?

Yes ,I have done it with the very latest release by paketo-buildpacks (2 weeks ago one )


The command I ran 
pack build my-app --builder custom-one --env BP_DOTNET_PROJECT_PATH=(where my .csproj file is located)
My builder consists of .net ,java and python .
What does my .net buildpack contain ?
It contains this ->https://github.com/paketo-buildpacks/dotnet-core/releases/tag/v0.42.2

![Uploading image.png…]()


Sorry for the late reply though. This one is causing headache for me now

**Comment by AshwinSridharan0410** [comment]
And this error is seen in azure functions related dotnet projects
